advanced/scanningArm/executeTrajectory.m

```diff
@@ -0,0 +1,149 @@
+function [ cmd, cmdIO, otherInfo, abortFlag ] = executeTrajectory( armGroup, ...
+                                        phoneGroup, ioGroup, cmd, cmdIO, ...
+                                        kin, traj, otherInfo )
+%EXECUTETRAJECTORY Plays out a trajectory similar to how the old blocking
```

Comment:
Maybe a bit more explanation of what this does - it's certainly a lot more than the old one. If it were part of the API, this could even be a separate class.